src/Portal/Portal.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks'
+
+import { Portal } from './index'
+
+<Meta
+  title="Components/Portal"
```

Comment:
Do we want this under some sort of "utilities" folder, or something like it?  Some way to distinguish that it's not necessarily intended to be used in the same way other components are?

Reply:
We could move it into a Utility folder in Storybook.

Reply:
Moved in cde3e24